<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Estudiantes</class>
 <widget class="QDialog" name="Frm_Estudiantes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insertar Estudiantes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/prefix1/13.png</normaloff>:/new/prefix1/13.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Nombre del Grupo :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_frmEstudiate_NomGup"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nombre :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>CI :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Edad :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sexo :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="LE_frmEstudiante_Nombre"/>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_frmEstudiante_CI">
           <property name="maxLength">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SB_frmEstudiante_Edad">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_frmEstudiante_Sexo">
           <item>
            <property name="text">
             <string>Masculino</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Femenino</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Calculo :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>EF :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Fisica :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Historia :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>EP :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Notas :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Nota_Calculo"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Nota_EF"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Nota_Fisica"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Nota_Historia"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Nota_EP"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Ausencias :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Ausencia_Calculo"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Ausencia_EF"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Ausencia_Fisica"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Ausencia_Historia"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Ausencia_EP"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/new/prefix1/f.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/iconos de menu/Sin título-2.png</normaloff>:/new/prefix1/iconos de menu/Sin título-2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_2">
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/iconos de menu/Sin título-1.png</normaloff>:/new/prefix1/iconos de menu/Sin título-1.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Frm_Estudiantes</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
